vending-app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,8 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="com.google.android.gms.permission.READ_SETTINGS" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES"/>
+    <uses-permission android:name="android.permission.READ_PRIVILEGED_PHONE_STATE"/>
```

Comment:
Is `android.permission.READ_PRIVILEGED_PHONE_STATE` really needed?
I think most microG users won't like this, the main intent of microG is to spoof data specially the IMEI.